pkg/plugins/resources/gitea/pullrequest/create_test.go

```diff
@@ -0,0 +1,72 @@
+package pullrequest
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreatePullRequest(t *testing.T) {
+
+	// Disable source testing with running short tes
+	if testing.Short() {
+		return
+	}
+
+	tests := []struct {
+		name     string
+		manifest struct {
+			URL          string
+			Token        string
+			Owner        string
+			Repository   string
+			SourceBranch string
+			TargetBranch string
+		}
+		wantResult string
+		wantErr    bool
+	}{
+		{
+			name: "pullrequest shouldn't be created on olblak/updatecli should not exist",
+			manifest: struct {
+				URL          string
+				Token        string
+				Owner        string
+				Repository   string
+				SourceBranch string
+				TargetBranch string
+			}{
+				URL:          "try.gitea.io",
```

Comment:
try.gitea.io should not be used for tests as it's not guaranteed to be working at all times or even preserve repositories there. Please use something more stable like codeberg.org

Reply:
Thanks for the feedback, `try.gitea.io` was a "temporary" solution until I find a better solution and then I discovered codeberg.org

I recently created an account over ther and now I have to migrate the tests